about.html

```diff
@@ -0,0 +1,114 @@
+                    <li class="list">N.I.E.S '22</li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <div id="hamburger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+    </div>
+    <main id="about-main">
+        <section id="top-section">
+            <p class="first-para">"Promoting Good Electrical Power For A Better Nigeria"</p>
+            <h1 class="topic">The Nigerian International Energy Summit 2022</h1>
+            <div class="hr1"></div>
+            <p class="second-para">In the spirit of the global agenda for energy transition and the need for us to remain reelevant and ahead, The Nigerian International Petroleum Summit quietly did a rebrand. From this moment, it'll be known as The Nigerian International Energy Summit. <br> With the ongoing reforms in the sector and improved engagement with host communities, Nigeria is positioning herself to be the foremost energy investment destination. The official Nigerian International Energy Summit continues to create the perfect platform for the world to meet Nigeria's energy industry players.</p>
+            <p class="third-para">For further information about N.I.E.S 2022, CONTACT US ON: <a href="mailto:iyaraogaga69@gmail.com"><b>iyaraogaga69@gmail.com</b></a></p>
+        </section>
+        <section id="mid-section">
+            <h6 class="heading1">Nigerian International Energy Summit 2022 Logo</h6>
+            <div class="hr2"></div>
+            <p class="fourth-para">The logo for the N.I.E.S 2022 was decided through the logo competition from 8 June to 7 July 2021.</p>
+            <div id="nig-logo3">
+                <img src="./Images/nigeria.png" class="ng" alt="nigerian map">
+                <h2 class="summit">N.I.E.S</h2>
+            </div>
+        </section>
+        <section id="bottom-section">
+            <h2 class="heading2">See The Past N.I.E Summits</h2>
+            <div class="hr1"></div>
+            <p class="fifth-para">Take a look at the last two N.I.E.S which took place in Lagos and Calabar, Nigeria.</p>
+            <div class="image-div">
+                <div class="first-image">
+                    <div class="orangebg">
+                    <a href="">
+                        <h3 class="heading3">2021</h3>
+                        <p class="sixth-para">Nigerian International Energy Summit 2021</p>
+                    </a>
+                    </div>
+                </div>
+                <div class="second-image">
+                    <div class="orangebg">
+                    <a href="">
+                        <h3 class="heading3">2020</h3>
+                        <p class="sixth-para">Nigerian International Energy Summit 2020</p>
+                    </a>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="partners">
+            <h3 class="ninth-header">Partners</h3>
```

Comment:
- [ ] Background color of this section should be `#272A31. Based on project requirements it is necessary to use basic color swatch as provided in the theme: 
<img width="319" alt="Screenshot 2022-04-13 at 17 44 06" src="https://user-images.githubusercontent.com/77038610/163194199-3872a2ee-6ed1-432f-bc3e-57baef3e4c70.png">

- [ ] Please make sure to use same colors in the project. 
- [ ] You should also be using same fonts,  which are `Cocogoose, Lato`:
<img width="1045" alt="Screenshot 2022-04-13 at 17 45 41" src="https://user-images.githubusercontent.com/77038610/163194544-10e000d2-c657-4017-9205-5b11253cc1d8.png">